descriptor.md

```diff
@@ -28,13 +28,14 @@ The following describe the primary set of properties that make up a _Descriptor_
   This property exists so that a client will have an expected size for the content before validating.
   If the length of the retrieved content does not match the specified length, the content SHOULD NOT be trusted.
 
-### Reserved
+- **`urls`** *array*
 
-The following are field keys that MUST NOT be used in descriptors specified in other OCI specifications:
+  This OPTIONAL property specifies a list of URLs from which this object can be downloaded.
+  For an ordinary layer, this property is empty and the layer contents can be retrieved directly from the registry.
 
-- **`urls`** *array*
+### Reserved
 
-  This key is RESERVED for future versions of the specification.
+The following are field keys that MUST NOT be used in descriptors specified in other OCI specifications:
```

Comment:
On Wed, Aug 24, 2016 at 10:47:20AM -0700, Antonio Murdaca wrote:
> >  
> > -  This key is RESERVED for future versions of the specification.
> > +The following are field keys that MUST NOT be used in descriptors specified in other OCI specifications:
> 
> *this is not

Ah, I see.  It was inherited from the parent's `### Reserved` header,
which this commit moves down here.  In that case, I'm fine with this
hunk as it stands, and will open a separate PR to address the wording
once this one lands.